lib/DDG/Spice/SeatGeek/EventsNearMe.pm

```diff
@@ -0,0 +1,40 @@
+package DDG::Spice::SeatGeek::EventsNearMe;
+# ABSTRACT: Returns upcoming concerts based on geolocation
+
+use DDG::Spice;
+
+primary_example_queries "live shows near me", "upcoming concerts in my area";
+description "Upcoming concerts from SeatGeek";
+name "SeatGeek Events By Geolocation";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/SeatGeek/EventsNearMe.pm";
+category "entertainment";
+topics "entertainment", "music";
+attribution github => ['https://github.com/MariagraziaAlastra','MariagraziaAlastra'],
+    github => ['https://github.com/andrey-p','Andrey Pissantchev'];
+
+triggers start => 'upcoming concerts',
+    'concerts',
+    'live',
+    'live shows',
+    'shows',
+    'gigs';
+
+spice proxy_cache_valid => "418 1d";
+spice is_cached => 0;
+
+spice to => 'http://api.seatgeek.com/2/events?taxonomies.name=concert&lat=$1&lon=$2&range=10mi&&callback={{callback}}';
```

Comment:
I'd prefer this to have either the default range (30mi) or a little wider. In some places there's a quite a lot of distance between cities and people might be interested in concerts in those near cities or regions as well, especially if they live in some little city which usually doesn't have lots of music events :)